modules/reactor/doc/content/source/meshgenerators/FlexiblePatternGenerator.md

```diff
@@ -10,7 +10,7 @@
 
 Instead of the fixed hexagonal and cartesian background shapes provided respectively by `PatternedHexMeshGenerator` and `PatternedCartesianMeshGenerator`, `FlexiblePatternGenerator` allows adoption of a closed curve of any type, provided by [!param](/Mesh/FlexiblePatternGenerator/boundary_mesh), to be used as the external boundary of the background region, with [!param](/Mesh/FlexiblePatternGenerator/boundary_type) set as `CUSTOM`. The [!param](/Mesh/FlexiblePatternGenerator/boundary_mesh) needs to be a mesh containing a single closed curve that resides in the XY plane and consists of only `EDGE2` elements. Examples of such curve meshes include polygon curves generated by [`PolyLineMeshGenerator`](PolyLineMeshGenerator.md), more generalized curves generated by [`ParsedCurveGenerator`](ParsedCurveGenerator.md), and even externally generated curves through [`FileMeshGenerator`](FileMeshGenerator.md).
 
-If a hexagonal, Cartesian, or circular external boundary are desired, [!param](/Mesh/FlexiblePatternGenerator/boundary_type) should be set as as `HEXAGON`, `CARTESIAN`, and `CIRCLE`, respectively. In that case, [!param](/Mesh/FlexiblePatternGenerator/boundary_size) and [!param](/Mesh/FlexiblePatternGenerator/boundary_sectors) also need to be specified. By default, the external boundary will be assigned an ID of `10000` to be compatible with other Reactor Module tools. Users can also define a custom id for the external boundary using [!param](/Mesh/FlexiblePatternGenerator/external_boundary_id).
+If a hexagonal, Cartesian, or circular external boundary are desired, [!param](/Mesh/FlexiblePatternGenerator/boundary_type) should be set as as `HEXAGON`, `CARTESIAN`, and `CIRCLE`, respectively. In that case, [!param](/Mesh/FlexiblePatternGenerator/boundary_size) and [!param](/Mesh/FlexiblePatternGenerator/boundary_sectors) also need to be specified. By default, the external boundary will be assigned an ID of `10000` to be compatible with other Reactor Module tools. Users can also define a custom id and name for the external boundary by using [!param](/Mesh/FlexiblePatternGenerator/external_boundary_id) and [!param](/Mesh/FlexiblePatternGenerator/external_boundary_name), respectively.
```

Comment:
it would be good to refer to a listing in some sort of ReactorDefaultNumbering.h header so that these numbers remain accurate

Reply:
Is there an example in the documentation of how to refer to actual values like 10000 from a heading file?

Reply:
use !listing

we do this from NS.h in the Navier Stokes doco

Reply:
Thanks will check it out

Reply:
boundary id 10000 is set in PolygonMeshGeneratorBase.h, which is now linked in the documentation